app/forms/linking_case/invitation_form.rb

```diff
@@ -0,0 +1,9 @@
+module LinkingCase
```

Comment:
To follow the pattern for copy_case and inline with controller naming this module should be called `LinkCase`